data/yam/agama/auto/lib/storage.libsonnet

```diff
@@ -0,0 +1,53 @@
+{
+  lvm(encrypted=false): {
+    drives: [
+      {
+        alias: 'pvs-disk',
+        partitions: [
+          { search: "*", delete: true }
+        ]
+      },
+    ],
+    volumeGroups: [
+      {
+        name: 'system',
+        physicalVolumes: [
+          {
+            [if encrypted == true then 'generate']: {
+              targetDevices: ['pvs-disk'],
+              encryption: {
+                luks2: { password: "nots3cr3t" }
+              }
+            },
+            [if encrypted == false then 'generate']: ['pvs-disk'],
+          },
+        ],
+        logicalVolumes: [
+          { generate: 'default' },
+        ],
+      },
+    ]
+  },
+  root_filesystem_ext4: {
+    drives: [
+      {
+        partitions: [
+          { search: "*", delete: true },
+          { generate: 'default' },
+          { filesystem: { path: '/', type: 'ext4' } },
```

Comment:
this could be a way to reuse it to avoid repetition below, passing the value from the openQA variable with the list of options.